### ISSUE TITLE
Build Warnings

### ISSUE THREAD

**Issue by john-** [ZhongHuaRong/LinPhone-Qt-Demo#1]
Hello,

Thanks for making this Linphone-Qt-Demo available.    Once I get it running on linux it will help me quite a bit.

I can successfully build both desktop-demo and quick-demo.   However, I cannot yet get video call to work.   I do get around 19,000 compiler warnings.   Here is one example:
`
/usr/local/src/LinPhone-Qt-Demo/desktop-demo/sdk/linux/linphone-sdk/desktop/include/linphone++/logging_service_listener.hh:52: warning: unused parameter ‘logService’ [-Wunused-parameter]
In file included from ../desktop-demo/sdk/linux/linphone-sdk/desktop/include/linphone++/linphone.hh:24,
                 from ../desktop-demo/src/app/paths/Paths.cpp:21:
../desktop-demo/sdk/linux/linphone-sdk/desktop/include/linphone++/logging_service_listener.hh: In member function ‘virtual void linphone::LoggingServiceListener::onLogMessageWritten(const std::shared_ptr<linphone::LoggingService>&, const string&, linphone::LogLevel, const string&)’:
../desktop-demo/sdk/linux/linphone-sdk/desktop/include/linphone++/logging_service_listener.hh:52:127: warning: unused parameter ‘logService’ [-Wunused-parameter]
   52 |                         LINPHONECXX_PUBLIC virtual void onLogMessageWritten(const std::shared_ptr<linphone::LoggingService> & logService, const std::string & domain, linphone::LogLevel level, const std::string & message) {};
      |                                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~
`
All of the 19,000 warnings are for other "unused parameters".   Any idea what is causing this?

Thanks,

John

**Comment by ZhongHuaRong** [comment]
The video is not available, maybe it was not selected when compiling linphone? You can refer to the compilation options in https://github.com/ZhongHuaRong/LinPhone-Qt-Demo/blob/main/doc/build.md, QT_GL is used For rendering video in qt, you must select

As for the warning message, you can try to add the following two lines to the pro file:
QMAKE_CXXFLAGS += -Wno-unused-parameter
QMAKE_CXXFLAGS += -Wno-unused-variable
But I haven’t verified this myself

**Comment by john-** [comment]
Thanks for pointing out the doc/build.md file.   As you indicate setting QT_GL was required.   In addition to that, I needed to make sure the openh264 and gogl mediastreamer plugins were available to the application at run time.   I did this with a Run Deployment step to copy the files to the application's location.  I can now place video calls!

I will work with the compiler switches to see if they disable the warnings.   As things are working though I will close this ticket.

Thanks again for making this available.   It is helping me quite a bit in understanding how to integrate linphone SDK,
